addons/udes_stock_picking_batch/models/stock_move_line.py

```diff
@@ -0,0 +1,128 @@
+from collections import defaultdict
+from datetime import datetime
+import logging
+
+from odoo import models, fields, api, _
+from odoo.tools.float_utils import float_compare, float_round
+from odoo.osv import expression
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    def _determine_priority_skipped_moveline(
+        self, skipped_product_ids=None, skipped_move_line_ids=None
+    ):
+        """Returns a priority move line based on the first move line found
```

Comment:
Nit: can we rephrase a bit and give some context about how skipped move lines are ordered? The docstring is phrased a bit weirdly.

Reply:
@armandcela Will resolve and suggest we look into this later as part of another story as you are time bounded right now.